modules/check_in/spec/serializers/vaos/vaos_appointment_data_serializer_spec.rb

```diff
@@ -0,0 +1,424 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CheckIn::VAOS::VAOSAppointmentDataSerializer do
+  subject { described_class }
+
+  let(:vaos_appointment_data) do
+    {
+      data: [
```

Comment:
do we really need to put the whole payload here to test the serializer, or can we trim it?

Reply:
Thought of having the real sample data received from VAOS service

Reply:
there's almost 300 lines of the payload, that are IMHO unnecessary. I would suggest trimming it and keeping the parts that are really needed to test the serializer and trimming the rest.

Reply:
Trimmed test data further to reduce the line of code.